packages/cli/src/cliEntry.js

```diff
@@ -70,16 +70,15 @@ function printHelpInformation() {
 }
 
 function printUnknownCommand(cmdName) {
-  logger.error(
-    [
-      cmdName
-        ? `Unrecognized command "${cmdName}".`
-        : "You didn't pass any command.",
-      `Run ${chalk.white(
-        '"react-native --help"'
-      )} to see list of all available commands.`,
-    ].join(' ')
-  );
+  cmdName
+    ? logger.error(
+        [
+          `Run ${chalk.white(
```

Comment:
You missed the unrecognized command message, can you add it back?

Reply:
Added back, thanks for noticing this!

I also found an issue with this part, if you do `cli --unknown-command` it will be parsed by commander but `cli unknown-command` will be handled by this function which will lead to two different error messages.

Reply:
>I also found an issue with this part, if you do `cli --unknown-command` it will be parsed by commander but `cli unknown-command` will be handled by this function which will lead to two different error messages.

We're going to rewrite `commander` to `yargs` soon, so let's not bother this inconsistency for now :)